test/testdrive-old-kafka-src-syntax/load-generator-key-value.td

```diff
@@ -0,0 +1,298 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
This one is longer because it first adds test coverage that wasn't there before, right?

Reply:
It doesn't hurt much either way, but no need to add a separate file here if there is nothing interesting being exercised. All of these `old-kafka-src-syntax` tests was supposed to be long removed, but then the new `CREATE TABLE ... FROM SOURCE` syntax work was paused for a while.